content/docs/user-guide/testing/parallel-pattern/_index.md

```diff
@@ -0,0 +1,526 @@
+---
+description: ' Use parallel pattern to allow tests to be run in parallel. '
+title: Parallel Pattern & Batching Test-Writing
+---
+
+## How editor automated tests work
+
+O3DE uses [CTest](https://cmake.org/cmake/help/latest/manual/ctest.1.html) to drive all testing across the engine. Python is used for automating the testing of the engine.
+
+A Python automated test consists of two Python interpreter instances:
+
+* **External Python interpreter**: The test driver for the test suite. It is responsible for launching the Editor and providing the tests to run to the embedded interpreter. This instance uses the [pytest](https://docs.pytest.org/) framework and LyTestTools for specifying the tests. LyTestTools is a set of Python tools and utilities that have been written around pytest.
+* **Embedded editor Python interpreter** (Hydra): This is an interpreter instance inside the Editor that is provided with bindings to interact with it. This instance is responsible for performing the actual test and is part of the EditorPythonBindings gem. These bindings are commonly known as Hydra.
+
+Upon test completion, the editor will be closed with an exit code depending on the result of the test (0 for success and 0xF for failure - other return codes will be set by the OS if the editor crashed).
+
+Both instances use the same Python executable found in *o3de/python/*
+
+This is the basic flow of a Python test:
+![Python Test Flow](images/user-guide/testing/parallel-pattern/automated-editor-test.png)
+
+## Test Structure
+
+Automated tests are composed of:
+
+* **Test Suite**:  A test suite is a list of tests that belong to a specific feature.
```

Comment:
I think it would be good to point out that the "TestAutomation" test suite must inherit from the "EditorTestSuite" class